scripts/prebuild.js

```diff
@@ -529,8 +535,7 @@ async function getSourceByName(api_key, source_name) {
       credentials: 'include',
     });
     if (response.status !== 200) {
-      manageResponseError(response);
-      return null;
+      manageResponseError(`Error getting source '${source_name}'`, response);
```

Comment:
@jmnavarro - this used to `return` so the flow has changed.

Reply:
Actually there are a few places where the flow has changed from either returning or throwing errors. I just want to make sure the behavior is intended.